Classes/ContextMenu/ItemProviders/FileProvider.php

```diff
@@ -16,55 +16,19 @@
 
 namespace Causal\FalProtect\ContextMenu\ItemProviders;
 
-use TYPO3\CMS\Core\Resource\Folder;
-use TYPO3\CMS\Core\Resource\FolderInterface;
+use Causal\FalProtect\Service\SecurityService;
```

Comment:
Too many changes in class FileProvider. It was supposed to be lightweight and reuse \TYPO3\CMS\Filelist\ContextMenu\ItemProviders\FileProvider but now it's totally different, and should better not extend \TYPO3\CMS\Filelist\ContextMenu\ItemProviders\FileProvider at all anymore.